cmplibrary/src/main/java/com/sourcepoint/cmplibrary/util/ViewsManager.kt

```diff
@@ -164,14 +168,27 @@ private class ViewsManagerImpl(
                     messSubCat = messSubCat,
                     viewId = cmpViewId
                 )
+                this.webview = newWebView
+                newWebView
             }
         } ?: Either.Left(WebViewCreationException(description = "The activity reference in the ViewManager is null!!!"))
     }
-
     override fun dispose() {
+        webview = null
         weakReference.clear()
     }
 
+    override fun onBackPressed() {
+        webview.let {
+            (webview as ConsentWebView).evaluateJavascript(
+                """
+                window.postMessage({ name: 'sp.BACK' })
+                """.trimIndent(),
+                null
+            )
+        }
+    }
```

Comment:
@DmitriySPUSA @bohdan-go-wombat why do we need to cast to `ConsentWebView`? Both classes respond to `evaluateJavascript`.

Reply:
@andresilveirah good point, since `webview` is a type of nullable `ConsentWebView`, there is no point in that extra check, so new thing should look something like:
```
override fun onBackPressed() {
    webview?.evaluateJavascript(
        """
            window.postMessage({ name: 'sp.BACK' })
            """.trimIndent(),
        null
    )
}
```

Reply:
@andresilveirah actually, we have to cast it to `ConsentWebView`, since our `webview` is a type of `IConsentWebView`, which is a custom interface that has no `evaluateJavascript` method in it's disposal